BasicLayoutsCodelab/app/src/main/java/com/codelab/basiclayouts/MainActivity.kt

```diff
@@ -119,12 +122,12 @@ fun AlignYourBodyElement(
                 .size(88.dp)
                 .clip(CircleShape)
         )
+        Spacer(Modifier.height(8.dp))
         Text(
             text = stringResource(text),
-            style = MaterialTheme.typography.h3,
-            modifier = Modifier.paddingFromBaseline(
-                top = 24.dp, bottom = 8.dp
-            )
+            modifier = Modifier.paddingFromBaseline(top = 8.dp),
```

Comment:
Is this method explained in the codelab? If so, we should make sure that it's actually this paddingFromBaseline that adds the space, not the Spacer that you added above.